org.eclipse.winery.yaml.converter/src/main/java/org/eclipse/winery/yaml/converter/yaml/Y2XConverter.java

```diff
@@ -203,6 +249,20 @@ private TServiceTemplate convertServiceTemplate(org.eclipse.winery.model.tosca.y
         }
 
         if (!node.getProperties().isEmpty()) {
+            // TODO Hack Property definitions
+            WinerysPropertiesDefinition properties = new WinerysPropertiesDefinition();
+            PropertyDefinitionKVList propertyDefinitionKVList = new PropertyDefinitionKVList();
+            node.getProperties().forEach((name, property) -> {
+                QName type = TypeConverter.INSTANCE.convert(property.getType());
+                propertyDefinitionKVList.add(new PropertyDefinitionKV(name,
+                    Optional.ofNullable(type.getPrefix())
+                        .map(i -> i.concat(":")).orElse("")
+                        .concat(type.getLocalPart())));
+            });
+            WinerysPropertiesDefinition propertiesDefinition = new WinerysPropertiesDefinition();
```

Comment:
This is a duplicate. You already introduced a `WineryPropertiesDefinition` variable `properties`  at the top of the `if` which is unused.